src/cmdliner.mli

```diff
@@ -244,6 +244,11 @@ module Term : sig
   (** [choice_names] is a term that evaluates to the names of the terms
       to choose from. *)
 
+  val capture : 'a t -> ('a * string list) t
```

Comment:
I would like to find a better name for this. `Term.capture t` is a bit odd to read: we are not capturing the term but the arguments it concretely parsed. 

Here are a few ideas `Term.{captured,parsed,actual,concrete}[_args}`. This might still be confused with accessing the term's actual specification though. Maybe we should throw an `eval` in there. Something like `Term.captured_eval_args`, bof, or maybe `Term.with_evaluated_args` ?

Reply:
`with_evaluated_args` seems good

Reply:
I wonder if it doesn't clash with the evaluation of a term though, the documentation looks a bit odd: `... is a term whose evaluation returns the arguments evaluated ...`. Other suggestions: `with_user_specified_args`, `with_args_from_command_line`.